Fix/mocha reporter

josdejong opened the pull request:
When running the almost ~5000 or so unit tests, a log line is outputted for every unit test. This gives huge log files, where you cannot see any "real" outputs because they're lost in the noise. 

This PR changes the reporter of Mocha to `dot`, outputting only a dot for every executed test.

Does that make sense to you @gwhitney ?

gwhitney commented:
Yes, it does make sense in general. But does it still report the fails in the long form? I will admit I usually debug with print statements (not being familiar with a good proper javascript debugger) and so it's important to locate the prints relative to the point of failure. So either if it reports errors in full or if there's a command-line flag to go back to the fuller output, that would be great

josdejong commented:
Thanks for thinking along.

> But does it still report the fails in the long form?

Yes all is reported, only thing is that inline console logs are less easy to locate since they are not proceeded by the title of the test. Here an example where I added a console.log in a unit test and changed a number in the expected output:

![afbeelding](https://user-images.githubusercontent.com/568626/162718913-bf21ad6b-f566-49aa-8ae0-0e586acf9740.png)


BTW, you can always switch to the extensive reporter if needed if you want the "old" output:
```
npm run test:src -- --reporter=spec
```
